services/frontend-v3/src/i18n/fr_CA.json

```
@@ -333,12 +333,14 @@
   "setup.talent.management.title": "Gestion de talents",
   "setup.teams.placeholder": "appuyez sur la touche d'entrée pour ajouter",
   "setup.welcome": "Bienvenue",
-  "setup.welcome.action": "Veuillez sélectionner un profil SAGE pour automatiquement remplir vos informations ou commencer de zéro",
+  "setup.welcome.action": "Veuillez sélectionner un profil GCannuaire pour automatiquement remplir votre organisation et vos coordonnonées ou vous pouvez commencer de zéro et synchroniser avec GCannuaire plus tard",
   "setup.welcome.description": "Nous avons besoin de quelques informations pour configurer votre profil",
-  "setup.welcome.geds.description": "de SAGE",
+  "setup.welcome.geds.description": "de GCannuaire",
   "setup.welcome.geds.title": "Récupération de profils",
   "setup.welcome.new.description": "débuter",
   "setup.welcome.new.title": "Nouveau utilisateur",
+  "setup.welcome.skip": "Sauter la création de profil ...",
```

I would personally remove the '...' 
```suggestion
  "setup.welcome.skip": "Sauter la création de profil",
```